### PR TITLE
Removed redundant find scripts 

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

Removed find scripts already offered by Cmake from isis/cmake directory. FindX11 was left in since the cmake version tries to find a dozen different x11 related libraries which can cause problems as they are all tagged as required when ISIS only wants libx11. 

## Related Issue

https://github.com/USGS-Astrogeology/ISIS3/issues/3360
## Motivation and Context

See issue

## How Has This Been Tested?

Built and ran tests on Linux box. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/CMakeLists.txt [hunk 1]
```diff
@@ -283,9 +314,6 @@ endforeach()
 foreach (_variableName ${_variableNames})
    get_filename_component(LIBDIR "${${_variableName}}" DIRECTORY)
    if (_variableName MATCHES "^CMAKE+")
-   elseif (_variableName MATCHES ".+_LIB$")
-     list(APPEND ALLLIBDIRS "${LIBDIR}")
-     list(APPEND ALLLIBS "${${_variableName}}")
```
Comment (Kelvinrr) [inline isis/CMakeLists.txt:288]: This if-statement wasn't really being used before, after switching to stock FindBoost.cmake, it caused vars internal to that script to slip through, so I simply removed it.

**Review by jessemapel — DISMISSED** [review]

File: isis/CMakeLists.txt [hunk 2]
```diff
@@ -219,11 +219,43 @@ find_package(Qt5 COMPONENTS
                 # Search this path explicitly for MacOS OpenGL Framework
                 PATHS /System/Library/Frameworks/ REQUIRED)
 
+
+set(Boost_USE_DEBUG_LIBS OFF)
+
 # Some of these will have non-traditional installs with version numbers in the paths in v007
 # For these, we pass in a version number, and use it in the path suffix
 # This only applies to v007, and outside of the building, we should only expect standard installs
 # The v007-specific installs are listed beside their find_package calls below:
```
Comment (jessemapel) [inline isis/CMakeLists.txt]: Can you remove this comment about V007 as we no longer support it.

Comment (Kelvinrr) [reply isis/CMakeLists.txt]: Good catch forgot to remove it.

**Comment by Kelvinrr** [comment]
Not sure why it's saying I dismissed the review? Don't recall pressing anything other than resolve conversation for Jesse's comment since I addressed it.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**